### ISSUE TITLE
CNN2 Problem with big datasets

### ISSUE THREAD

**Issue by DanielCoelho112** [AutomecUA/AutoMec-AD#27]
Arrays sizes exceed available RAM.
Possible Solution: Transfer Learning, this way, we would train the same model multiple times but using different datasets

**Comment by DanielCoelho112** [comment]
Problem solved.
I implemented the transfer learning in the script "train2a_transfer_learning.py".
Please, somebody test to see if works on any PC besides mine. The var "n_times_tf" represents the number of times that the training will occur, currently is 10, but is supposed to change depending on the dataset used.

**Comment by manuelgitgomes** [comment]
I tried to run it. It gave me an odd output. See if I have done someting wrong. I used the training2a_tf.launch to run it. It gave me this output: 
![Screenshot from 2021-08-24 15-01-12](https://user-images.githubusercontent.com/58152228/130630807-35bf9ce8-659b-4710-a542-6da5ccb24dca.png)

**Comment by DanielCoelho112** [comment]
@manuelgitgomes usually, this odd output occurs when you forget to add the extension .h5 to the model name. Can you try again?

**Comment by manuelgitgomes** [comment]
Found a bug, might not be very useful, but when number of right/left angles are 0, code breaks due to dividing by 0
![Screenshot from 2021-09-10 00-04-48](https://user-images.githubusercontent.com/58152228/132773942-3aaf67ee-29a0-4682-8dd7-237b645ed079.png)

**Comment by manuelgitgomes** [comment]
Nevertheless, it runs well!
![Screenshot from 2021-09-10 00-11-03](https://user-images.githubusercontent.com/58152228/132774259-ce50e557-9aac-43f7-b422-f516c5f96f45.png)

**Comment by manuelgitgomes** [comment]
Shouldn't this be marked as closed?

**Comment by DanielCoelho112** [comment]
I believe it is done, however, please review the pull request and then merge.

**Comment by manuelgitgomes** [comment]
Another thing we need to add is a launch file for this cnn on physical_bringup and simulation_bringup. I will take care of this now.

**Comment by DanielCoelho112** [comment]
That is already done. We only need a new launch for training (and that is already done in `traxxas_training2a_tf.launch` ) because, for the driving part, we should use the `traxxas_drive2a.launch`.

**Comment by manuelgitgomes** [comment]
Sorry, my bad. But I believe it is only done on simulation_bringup and not on physical_bringup. I know they are interchangeable, but for consistency's sake, I think they should be on both packages. Do you agree?

**Comment by DanielCoelho112** [comment]
Yes, I didn't notice that. I forgot to add the launch in physical_bringup. So please add the launch on physical_bringup and then I believe we can close this issue.

**Comment by manuelgitgomes** [comment]
Done on #69
